docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -47,7 +47,42 @@ uavcan.protocol.NodeStatus (Data Type ID = 341)
 The DroneCAN protocol requires that all nodes on a DroneCAN network periodically publish their status using the Node Status message. Vertiq modules support this behavior to conform 
 to the standard. A uavcan.protocol.NodeStatus message is published at 1 Hz during operation, reporting its health, current mode, and uptime.
 
-In normal operation a Vertiq module should always report its Health as OK, and its mode as Operational. No sub-modes or vendor specific status codes are currently supported.
+Vertiq's Node Health Indications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Vertiq modules support all four standard DroneCAN `health indications <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#:~:text=fatal%20malfunction.%0Auint2-,health,-%23%0A%23%20Current%20mode.%0A%23%0A%23%20Mode>`_. Modules enter each state given the following:
+
+Critical
+""""""""""
+#. The module's :ref:`total derate <derates>` reaches 0
+#. Your module's drive mode is in lockout due to :ref:`power safety <power_safety>`
+
+Warning
+""""""""""
+#. The module's supply voltage is greater than your drive's :ref:`v_max_start <brushless_drive>`. This typically indicates that the module is regenerating into a supply that cannot handle regenerated current, such as a typical benchtop power supply, and is limiting how much it regenerates to protect itself
```

Comment:
Linking to the safety system stuff (https://iqmotion.readthedocs.io/en/feature-update_dronecan_health_info/manual/manual_safety_systems.html) might be better here than just the table, or linking both could be good to. The safety system part just gives a lot more context on what the heck this all really means for a customer. And maybe it would be better to call this Volts Limit Starting Voltage instead of v_max_start for a more descriptive name and to match what's in Control Center and on the Safety Systems page. The other links here got the safety system links, so it's consistent as well